weblate/trans/tests/utils.py

```diff
@@ -254,10 +254,12 @@ def create_po_branch(self):
     def create_po_push(self):
         return self.create_po(push_on_commit=True)
 
-    def create_po_empty(self):
-        return self._create_component(
-            "po", "po-empty/*.po", new_base="po-empty/hello.pot", new_lang="add"
-        )
+    def create_po_empty(self, project=None):
+        kwargs = {"new_base": "po-empty/hello.pot", "new_lang": "add"}
+        if project:
+            kwargs |= {"project": project}
```

Comment:
Why not directly set it?

```suggestion
            kwargs["project"] = project
```